src/constants.ts

```diff
@@ -194,9 +194,13 @@ export class Constants {
         [TemplateLanguage.Ruby]: "ruby",
     };
 
+    // 'error', 'errorMessage' and 'stack' are used in vscode-azureextensionui npm

```

Comment:
Can we control the telemetry in vscode-azureextensionui package? Maybe we need to ask the maintainer for an update?

Reply:
We could not control the telemetry property in vscode-azureextensionui package. I don't think we need to ask them to change telemetry property name.